### ISSUE TITLE
Should take the mean of activations (ReLU), not Conv2d

### ISSUE THREAD

**Issue by mirkomiorelli** [MrinalJain17/vgg-normalized#1]
Nice work, however I think you might have a mistake in [here](https://github.com/MrinalJain17/vgg-normalized/blob/main/vgg_normalize.py#:~:text=x%20%3D%20layer,%2C%203). 

While in Keras/tensorflow the conv layer includes the activation function (ReLU), in Pytorch it does not. So effectively you are calculating the means of the filters before the activation in your code. If my understanding is correct, you should take the means after applying the activation function. 

Cheers!
